app/services/ocfl_migrator.rb

```diff
@@ -0,0 +1,43 @@
+# For a given druid, migrates files from the Purl and Stacks mounts to an OCFL structure
+class OcflMigrator
+  def self.migrate(...)
+    new(...).migrate
+  end
+
+  # @param [String] druid
+  def initialize(druid:)
+    @druid = druid
+  end
+
+  def migrate
+    builder.copy_recursive(stacks_druid_path)
+    builder.copy_recursive(purl_druid_path)
```

Comment:
Should `OcflMigrator.migrate` blow up if one or both of the Purl and Stacks paths are missing?

Reply:
No because Web archiving won't have files in `/stacks`